alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/CustomWebhook.kt

```diff
@@ -128,6 +128,7 @@ data class CustomWebhook(
 
         @JvmStatic
         @Throws(IOException::class)
+        @Suppress("UNCHECKED_CAST")
```

Comment:
NP: you still need this? suppressWarning

Reply:
Removed.